CrashReporter.xcodeproj/project.pbxproj

```diff
@@ -2574,6 +2599,7 @@
 				05E731F20EFA1AAB005EDFB7 /* CrashReporter macOS */,
 				05CD31510EE936A9000FDE88 /* CrashReporter iOS */,
 				8064D7AD1C4D22D8005A8B4C /* CrashReporter tvOS */,
+				F8B9C72A24695BE600B9FEF6 /* CrashReporter XCFramework */,
```

Comment:
`Disk Image` should depend on it and build one more zip archive. Since it will be run during building the main binaries, you can -
1. Set proper target dependencies
2. Remove additional step on CI and just adjust the current ones to copy one more zip